ibm/resource_ibm_pi_instance.go

```diff
@@ -656,6 +664,39 @@ func resourceIBMPIInstanceUpdate(ctx context.Context, d *schema.ResourceData, me
 		}
 	}
 
+	if d.HasChange(PIInstancePlacementGroupID) {
+
+		pgClient := st.NewIBMPIPlacementGroupClient(ctx, sess, cloudInstanceID)
+
+		oldRaw, newRaw := d.GetChange(PIInstancePlacementGroupID)
+		old := oldRaw.(string)
+		new := newRaw.(string)
+
+		if len(strings.TrimSpace(old)) > 0 && old != "none" {
+			placementGroupID := old
+			//remove server from old placement group
+			body := &models.PlacementGroupServer{
+				ID: &instanceID,
+			}
+			_, err := pgClient.DeleteMember(placementGroupID, body)
+			if err != nil {
+				return diag.Errorf("failed to perform the placement group delete operation... %v", err)
```

Comment:
Just return the err, `DeleteMember` has the proper message with wrapped err.

Reply:
I made this suggested change.